components/Groups/AddNewGroupButton.js

```diff
@@ -1,12 +1,21 @@
 import { useRouter } from "next/router";
+import { BsPlusCircle } from "react-icons/bs";
 
 function AddNewGroupButton() {
   const router = useRouter();
 
   function handleRedirect() {
     router.push("/groups/add-group");
   }
-  return <button onClick={handleRedirect}>Add new group</button>;
+  return (
+    <div className="add-button-container">
+      <button onClick={handleRedirect} className="add-new-group-button">
+        <BsPlusCircle className="add-new-icon" />
```

Comment:
Should we add an aria-label to that one?

Reply:
Good call, we should!